pages/1.13/security/oss/authentication/authentication-token/index.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: layout.pug
+navigationTitle: DC/OS Authentication Token
+title: DC/OS Authentication Token
+excerpt: Getting familiar with DC/OS Authentication tokens
+menuWeight: 10
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+DC/OS uses JSON Web Tokens for authenticating requests to the cluster. In DC/OS terminology, these are named `DC/OS Authentication tokens`. The Identity and Access Manager is the only entity in the cluster that issues DC/OS Authentication tokens.
+
+# DC/OS Authentication token format
+
+A DC/OS Authentication token is a RFC 7519 JSON Web Token (JWT) of type RS256. The JWT payload contains the user or service ID that the token was issued for in the (`uid`) claim and an (`exp`) claim indicating the time after which the token will be considered invalid.
+
+JWT Header
+```json
+{
+    "alg": "RS256",
+    "typ": "JWT"
+}
+```
+
+JWT Payload
+```json
+{
+    "uid": "<uid>",
+    "exp": <expiration_time>
+}
+```
+
+In the process of generating a new DC/OS Authentication token the IAM signs the token with its private key.
+
+DC/OS Authentication tokens can be inspected via [jwt.io](https://jwt.io) or via your favorite JWT library.
+
+# Lifetime and renewal
+
+In Open DC/OS Authentication tokens are valid for five days after they have been issued. After five days, you will need to log in again to obtain a new token. Using [service accounts](/1.13/security/oss/user-account-management/service-accounts/) and the [service login](/1.13/security/oss/login/service-login/) make it easy to automate this process when deploying long-running services.
```

Comment:
Replace "you will need to log in again...." with "you must log in again to ......"